client/web/src/index.css

```diff
@@ -31,6 +31,107 @@
   .card td:last-child {
     @apply text-neutral-800 text-sm leading-tight;
   }
+
```

Comment:
where did these all come from?  Is this out of adminpanel?  Should we be thinking about a better way to structure this, or is index.css going to become a giant file of a bunch of disparate component styles?

Reply:
Yeah this is from the admin panel. So I think the stuff below in this file we'll be able to get rid of once we get rid of legacy. But we could consider breaking up the file into a couple separate ones